modules/apps/object/object-api/src/main/java/com/liferay/object/field/util/ObjectFieldUtil.java

```diff
@@ -29,14 +29,14 @@ public class ObjectFieldUtil {
 
 	public static ObjectField createObjectField(
 		long listTypeDefinitionId, String businessType, String dbColumnName,
-		String dbType, boolean indexed, boolean indexedAsKeyword,
+		String dbType, String externalReferenceCode, boolean indexedAsKeyword,
```

Comment:
@seltong please reorder so that `externalReferenceCode` comes before the `listTypeDefinitionId`